src/components/Footer.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <footer>
+    <div class="footerLogos">
+      <img class="logo" src="../assets/refsay_logo.png" alt="Refugee say logo">
+      <img class="logo" src="../assets/ml_logo.png" alt="MIT media logo">
+      <img class="logo" src="../assets/oxford_logo.png" alt="Oxford logo">
+    </div>
+  </footer>
+</template>
+
+<script>
+export default {
+  name: 'Footer',
+};
+</script>
+
+<!-- Add "scoped" attribute to limit CSS to this component only -->
+<style scoped>
```

Comment:
there seems to be styles that go inside each other, what you think about using [SCSS](http://sass-lang.com/)?

```vue
<style lang="scss" scoped>
footer {
  /* .... */

  .footerLogos {
    /* .... */

    .logo {
      /* .... */
    }
  }
}
</style>
```
you need then also add `sass-loader` and `node-sass`
```bash
yarn add -D sass-loader node-sass
```